src/synthesized_insight/check.py

```diff
@@ -141,6 +141,13 @@ def affine(self, sr: pd.Series) -> bool:
             return True
         return False
 
+    def binary(self, sr: pd.Series) -> bool:
```

Comment:
we need to decide whether to leave this here or move it as a custom check

Reply:
Doing a custom check for now